test/RESTAssured/pom.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.backbase</groupId>
+  <artifactId>local-backend-setup-restassured</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <name>RESTAssured</name>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
+
+    <junit.version>4.13.2</junit.version>
+    <restAssured.version>5.3.0</restAssured.version>
+    <gson.version>2.10.1</gson.version>
+
+    <cucumber.version>4.7.1</cucumber.version>
+    <spring-boot.version>2.1.5.RELEASE</spring-boot.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>${junit.version}</version>
+      <scope>compile</scope>
```

Comment:
the compile is the default scope and can be removed, `test` is a better scope IMO

Reply:
Maybe this is my bad design. I have assertions in the pseudo service code, which is now in `main` rather than `test`.
Thanks for the tip on `compile` scope. Everything was originally in `test` before I moved some of the code over to main.
I can move all the code back into `test` if that's preferred, but it was hard to see just the tests

_(Pending a push with changes based on your other feedback)_ 
I have removed the `compile` scopes for the dependencies used in `main`

Reply:
I think the structure is fine, all test classes are in `test` folder and that makes it easy to integrate tests in a pipeline